scripts/bdm.py

```diff
@@ -0,0 +1,24 @@
+ # Module sys has to be imported:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [indentation is not a multiple of four (comment)](https://app.codacy.com/app/jorgeMFS/TMCompression/pullRequest?prid=4701004)